shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/NearbyStaticData.kt

```diff
@@ -344,7 +372,16 @@ data class NearbyStaticData(val data: List<TransitWithStops>) {
                 line = line,
                 routes = routes,
                 stop = stop,
-                patterns = linePatterns,
+                patterns =
+                    linePatterns.filter {
+                        when (it) {
+                            is StaticPatterns.ByDirection ->
+                                // Remove all directions terminating mid-line at Gov Center,
+                                // this direction label should never be displayed.
+                                it.direction.destination != "Government Center"
```

Comment:
I'm confused. Shouldn't even the patterns that end at Government Center be sorted into a "Gov Ctr & North" direction?

Reply:
Yeah, but by this point they will have already been sorted into the overridden "Gov Ctr & North" direction. The bare "Government Center" direction _only_ shows up when the direction has not been overridden by a grouped direction, which can only occur at or after the Gov Center stop. It's normal for it to happen at Gov Center for C and B trains, since the list of served stop IDs don't include any upstream stops that would override their default destination. After these changes, I think that should be the only place it occurs.

Reply:
Ah, if it's the presence of downstream stops that causes the override to kick in then that'd do it. It may be worth clarifying in the comment that trips continuing past Government Center will be grouped into "Gov Ctr & North" even at Government Center but trips terminating at Government Center will still say Government Center at Government Center. That's still confusing, because at Government Center shouldn't the trips that terminate at Government Center be hidden for being arrival-only?

Reply:
Yes, the trips are hidden, but this resolution is happening in the static data, so until we load the actual upcoming trips, we don't know whether or not there will be trips or if they should be hidden. And since these are typical patterns, this was causing the "Government Center" direction to flash in with "No realtime data" before any trips were loaded, at which point it would disappear. But yeah I'll add to the comment.

Reply:
Ah, that makes sense, since we know things are arrival-only by the schedules or the predictions. I think we could also determine that things are arrival-only in static data by checking the stop lists of representative trips, but if that's not simpler than this then there's no point doing that instead.

Reply:
I took a shot at expanding the comment, it's uhhhh, pretty verbose and potentially incoherent, but is hopefully more helpful.

Reply:
I think that helps.